### DISCUSSION TITLE
Duco - ESP serial kabel. Rol pin 3 weerstanden

### DISCUSSION THREAD

**Discussion by Maarten71** [arnemauer/Ducobox-ESPEasy-Plugin#154] [Q&A] [answered]
Ik heb een Wemos D1 mini V4 gekoppeld met de een CC1101 kaartje volgens de aansluitingen en heb de RF gateway werkend gekregen. Hierbij voed ik de D1 via zijn ESB poort. 

Nu de serial gateway: Als ik naar het schema van de Ventilation gateway kijk, dan zie ik een aparte 5V voeding en een kabel die via een 3,5mm jack een koppeling maakt met de Duco box.

De aparte kabel tussen de gateway en Ducobox is kennelijk een andere dan de USB-C kabel voor de Duco netwerk tool. Correct?

Onder het maken van de kabel (https://github.com/arnemauer/Ducobox-ESPEasy-Plugin/wiki/Hardware---Serial-Gateway) staat voor pin 3: (Duco box) 3 - Bootpin - (ESP) x	(Description) with 1K resistor connected to boot0 of STM32F0x0, boot0 is connected with 2K resistor to pin 5 (vcc). Ik begrijp de description niet. Moet ik deze weerstanden, aan de Duco zijde, aanbrengen of is dit ter info en moet ik alleen RX / TX en ground via de kabel verwerken.

**Comment by Maarten71** [comment]
@arnemauer, Arne kan jij helpen?

**Reply by arnemauer** [reply, marked as answer]
Pin 3 en 5 kun je negeren. Er staat een xje bij de kolom "esp".
Je hoeft alleen TX, RX en GND aan te sluiten.

**Reply by Maarten71** [reply]
Helder. Dank.
